package.json

```diff
@@ -63,7 +63,8 @@
     "regenerator-runtime": "~0.13.3",
     "router5": "~7.0.2",
     "router5-plugin-browser": "~7.0.2",
-    "store2": "~2.10.0"
+    "store2": "~2.10.0",
+    "react-windowed-select": "^2.0.1"
```

Comment:
It looks like we should *not* be specifying `react-select` as a direct dependency ourselves - i.e. we should remove it from this list. This new library specifies it as a dep (and notably not a peer-dep) with a particular version.  We only want one copy of `react-select` in our project and it should be the version this new library will work with.

Also, we want to use the `~` semver qualifier for our deps - this allows auto-upgrades of patch versions only, not minor versions.  (We have seen that minor versions are more likely to break, despite what the authors might claim with their version number choice.)

Finally, these deps should remain in alpha-order, and any time we change package.json we need to run `yarn` and check-in the modifications to `yarn.lock` as part of the change.